### PR TITLE
Portfolio

### PR DESCRIPTION
**Pull request by Tushar151997**
Overview
**Adding the HTML file to display the portfolio page
**Adding the CSS file for the styling
** Fixed the errors using linters

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+            <header>
+                <a href="#" class="logo">My Logo</a>
+                <nav class="ham">
+                <img src="images/header_img/Union.png" alt="bar">
+                </nav>
+            </header>
+            <section>
+                <div class="bg_image">
+                    <img class="background" src="images/header_img/header.png" alt="background_img">
+                </div>
```
Comment (ichala) [inline index.html]: - [ ] for aesthetic reasons the images shouldn't be part of the content, kindly use  `CSS background-image` instead , you can read more about it [here](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image)

File: index.html [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+</head>
+<body>
+    <div id="container">
+        <div id="template1">
+            <div class="bar">
+                <p>9:41</p>
+                <img class="mobile_network" src="images/header_img/tower.png" alt="network_img">
+                <img class="mobile_wifi" src="images/header_img/Wifi.png" alt="wifi_logo">
+                <img class="mobile_battery" src="images/header_img/Battery.png" alt="battery">
+            </div>
```
Comment (ichala) [inline index.html]: - [ ] Please note that this bar is not  part of the design and it shouldn't show up in your website , kindly remove it it only added on Figma to show you that the design is on mobile

image

File: index.html [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+                    <h3>LET'S CONNECT</h3>
+                </div>
+                <ul>
+                    <li><a href=""><img class="twitter" src="images/header_img//twiter.png" alt="twiter_logo"></a></li>
+                    <li><a href=""><img class="linkedIn" src="images/header_img//linkedin.png" alt="linkedIn_logo"></a></li>
+                    <li><a href=""><img class="m" src="images/header_img//m.png" alt="m_logo"></a></li>
+                    <li><a href=""><img class="github" src="images/header_img//github.png" alt="github_logo"></a></li>
+                    <li><a href=""><img class="ok" src="images/header_img//ok.png" alt="ok_logo"></a></li>
+                </ul>
+            </section>
```
Comment (ichala) [inline index.html]: - [ ] please add the missing border radius on the bottom left corner of the section to match Figma design

image

File: index.html [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+                <img class="mobile_wifi" src="images/header_img/Wifi.png" alt="wifi_logo">
+                <img class="mobile_battery" src="images/header_img/Battery.png" alt="battery">
+            </div>
+            <header>
+                <a href="#" class="logo">My Logo</a>
+                <nav class="ham">
+                <img src="images/header_img/Union.png" alt="bar">
+                </nav>
+            </header>
+            <section>
```
Comment (ichala) [inline index.html]: - [ ] all the content of the headline section should be aligned with flexbox, This gives us a lot of options to align all our elements, plus, it's responsive. so kindly fix it 🙏

File: index.html (hunk 3 above)
Comment (ichala) [inline index.html]: - [ ] Please note that the background color of the navbar & the headline section should be both white (`#ffffff`) and not `#f7f7f9` only the body of the website is using that color , kindly fix that

**Review by ichala — CHANGES_REQUESTED** [review]
Hi @Tushar151997 , 

Good job so far! 👏👏👏 
There are some issues that you still need to work on to go to the next project but you're almost there!

the-simpsons-homer-simpson

## To Highlight 🎉

- Good Redme✅ 
- Great Design✅ 
- Linters are passing ✅


 ## Required Changes ♻️

- [ ] **_🛑 Please Update your PR description & PR Title to be more descriptive, they should have a short summary describing introduced changes in more detail 🛑_** example:
- PR Title: Create navbar & header section
- PR Description : 
> - setup linters
> - create navbar
> - create a headline section
> - update readme 
> - etc


- [ ] _**check the comments below**_


## Optional suggestions

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+<body>
+  <header>
+    <a class="log">My Logo</a>
+    <nav>
+      <button class="header-button">
+        <span class="bar"></span>
+        <span class="bar"></span>
+        <span class="bar"></span>
+      </button>
+    </nav>
```
Comment (ichala) [inline index.html]: - [ ] please not that after the changes you made I cannot see that navbar button anymore , kindly fix that 

image

File: style.css [hunk 6]
```diff
@@ -0,0 +1,115 @@
...
+  margin: 16px 16px;
+  background-color: #f7f7f9;
+}
+
+.background-img {
+  background-image: url(./img/mobile-background.png);
+  background-size: contain;
+  background-repeat: no-repeat;
+  width: 100%;
+}
```
Comment (ichala) [inline style.css]: - [ ] as in the previous review please add the border-radius in the bottom left side of the headline section

**_Hint : you can add `border-bottom-left-radius: 50px;` to your section styling_**



image

**Review by ichala — CHANGES_REQUESTED** [review]
Hi @Tushar151997 , 

Good job so far! 👏👏👏 
There are some issues that you still need to work on to go to the next project but you're almost there!

the-simpsons-homer-simpson

## To Highlight 🎉

- Good Pr description ✅ 
- Good Design ✅
- Linters are passing ✅


 ## Required Changes ♻️

- [ ] **_Check the comments below_**



## Optional suggestions

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+        </p>
+        <div class="text">let's connect</div>
+        <ul class="button-bott">
+          <li><a href=""><img class="twitter" src="img/twiter.png" alt="twiter_logo"></a></li>
+          <li><a href=""><img class="linkedIn" src="img/linkedin.png" alt="linkedIn_logo"></a></li>
+          <li><a href=""><img class="m" src="img/m.png" alt="m_logo"></a></li>
+          <li><a href=""><img class="github" src="img/github.png" alt="github_logo"></a></li>
+          <li><a href=""><img class="ok" src="img/ok.png" alt="ok_logo"></a></li>
+        </ul>
+    </section>
```
Comment (fobadara) [inline index.html:43]: [REQUIRED] Due to some changes you now have two spaces on the left and right.

Your Design

image

[REQUIRED] Your `My Logo` is also moving to the next line now

**Review by fobadara — CHANGES_REQUESTED** [review]
Hi @Tushar151997, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which haven't been addressed yet.


 ## Highlights
✅ Linters are passing

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Tushar151997** [comment]
Thank you for the review please check I have made the required changes @fobadara

**Review by franclobo — APPROVED** [review]
Hi @Tushar151997, 

# STATUS: APPROVED 😃 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- No linter errors ✔️ 
- Correct Github flow ✔️ 
- Descriptive PR ✔️ 

## Optional suggestions
- [ ] **[OPTIONAL]** Kindly make sure that all the links in the README file work as expected. To fix the MIT link please 🙏🏽 add the MIT LICENSE file to your repository for a professional README file.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**